### PR TITLE
Fixed broken/incorrect links

### LINKED ISSUE #1002: Test and fix all README links (ui)

### PR DESCRIPTION
**Pull request by josh-peters-2357**
Went through whole readme and verified that each hyperlink goes to the correct destination.

resolves #1002

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,7 +94,7 @@ The following extensions are recommended for any code editor used to develop thi
 
 - [EditorConfig](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) ensures that indentation style, line endings, and file endings are consistent across editors and operating systems
 - [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) checks syntax correctness for JavaScript (`.js` files)
-- [Stylelint](https://marketplace.visualstudio.com/items?itemName=shinnn.stylelint) checks syntax correctness for Sass (`.scss` files)
+- [Stylelint](https://marketplace.visualstudio.com/items?itemName=glen-84.sass-lint) checks syntax correctness for Sass (`.scss` files)
```
Comment (EjPlatzer) [inline README.md:97]:
```suggestion
- [Stylelint](https://marketplace.visualstudio.com/items?itemName=stylelint.vscode-stylelint) checks syntax correctness for Sass (`.scss` files)
```
We use stylelint, not sass-lint, for linting our `.scss` files.

File: README.md [hunk 2]
```diff
@@ -22,10 +22,11 @@ This project is the frontend of Gordon 360 in React. [The retired frontend](http
 - [Continuous Integration and Deployment](#continuous-integration-and-deployment)
 - [Contributing](#contributing)
 - [Known Issues](#known-issues)
+- [History](#history)
 
 ## Getting Started
 
-This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app). Read the user guide [here](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md).
+This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app). Read the user guide [here](https://github.com/facebook/create-react-app/blob/master/packages/cra-template/template/README.md).
```
Comment (EjPlatzer) [inline README.md:29]:
```suggestion
This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app). Read the user guide [here](https://github.com/facebook/create-react-app/blob/master/packages/cra-template/template/README.md).
```
We should fix both CRA links on this line.

File: README.md [hunk 3]
```diff
@@ -57,7 +58,7 @@ After cloning this repo, and after any major changes to local code (like changin
 
 By default, React will use the live server backend to allow seamless front end development. If you would like to run the backend locally and connect to the UI repository, use the following steps:
 
-- After connecting to the virtual machine and setting up the backend, as documented in [gordon-360-api](https://github.com/gordon-cs/gordon-360-api/blob/develop/README.md#running-the-server-locally),
+- After connecting to the virtual machine and setting up the backend, as documented in [gordon-360-api](https://github.com/gordon-cs/gordon-360-api/blob/develop/README.md#machines-and-sites),
```
Comment (EjPlatzer) [inline README.md:61]:
```suggestion
- After connecting to the virtual machine and setting up the backend, as documented in [gordon-360-api](https://github.com/gordon-cs/gordon-360-api/blob/develop/README.md#running-the-api-locally),
```
I think this is where this link is meant to point to.

**Review by EjPlatzer — CHANGES_REQUESTED** [review]
Thanks for looking through this! I think a couple of the links are not quite right, so I've suggested some changes.

**Comment by josh-peters-2357** [comment]
Hey Evan! I made the suggested change. I rerequested review; I'm not sure if that's the right thing to do.

**Review by EjPlatzer — APPROVED** [review]
Looks great! Thanks for taking the time to check our links and fix everything up!

**Pull request merged**